packages/jest-matchers/src/index.js

```diff
@@ -136,12 +136,12 @@ const setState = (state: MatcherState) => {
 const getState = () => global[GLOBAL_STATE].state;
 
 // add default jest matchers
-addMatchers(matchers);
-addMatchers(spyMatchers);
-addMatchers(toThrowMatchers);
+expect.extend(matchers);
+expect.extend(spyMatchers);
+expect.extend(toThrowMatchers);
 
 module.exports = {
-  addMatchers,
+  addMatchers: expect.extend, // TODO: remove!
```

Comment:
TODO: My goal is to remove this export. Does that sound good? I just need to go throw and update any code that depends on it.